### ISSUE TITLE
Fiction live images error

### ISSUE THREAD

**Issue by i-push-code** [JimmXinu/FanFicFare#933]
https://fiction.live/stories/naive-schoolgirl-sakuras-exchange-trip/d8gx32QggbX8JYbET
This stories images arent showing up and Im using the latest fanficfare version

**Comment by JimmXinu** [comment]
Looking at it in the browser inspector, it looks like the HTML refers to images URLs like: 

`https://www.filepicker.io/api/file/7Fcd4vBTSeiB1t49RoxT`

...which fail, but then something in the javascript inside calls a URL like:

`https://cdn3.fiction.live/fp/7Fcd4vBTSeiB1t49RoxT?&quality=95`

Note the similarity of UID.  

filepicker.io appears to be a CDN of some kind.  The error I get in FFF is `429 Client Error: Too Many Requests for url:`

I don't even pretend to know how adapter_fictionlive works.  @HazelSh, do you have the time and interest to look at this?

**Comment by Jemeni11** [comment]
> Note the similarity of UID.

```py
if url.startswith("https://www.filepicker.io/api/"):
  logger.warning("Filepicker.io image detected, converting to Fiction.live image. This might fail.")
  url = f"https://cdn3.fiction.live/fp/{url.split('/')[-1]}?&quality=95"

```
And it worked for the ten images I tested it with.
I tested it in a standalone script however, so I do not know if this will work well with Fanficfare.

**Comment by HazelSh** [comment]
I'm ill with covid at the moment, can't promise to get to this soon.

**Comment by JimmXinu** [comment]
Ugh.  Hope you feel better soon.

**Comment by i-push-code** [comment]
> Looking at it in the browser inspector, it looks like the HTML refers to images URLs like:
> 
> `https://www.filepicker.io/api/file/7Fcd4vBTSeiB1t49RoxT`
> 
> ...which fail, but then something in the javascript inside calls a URL like:
> 
> `https://cdn3.fiction.live/fp/7Fcd4vBTSeiB1t49RoxT?&quality=95`
> 
> Note the similarity of UID.
> 
> filepicker.io appears to be a CDN of some kind. The error I get in FFF is `429 Client Error: Too Many Requests for url:`
> 
> I don't even pretend to know how adapter_fictionlive works. @HazelSh, do you have the time and interest to look at this?



> > Note the similarity of UID.
> 
> I came up with this:
> 
> ```python
> if url.startswith("https://www.filepicker.io/api/"):
>   logger.warning("Filepicker.io image detected, converting to Fiction.live image. This might fail.")
>   url = f"https://cdn3.fiction.live/fp/{url.split('/')[-1]}?&quality=95"
> ```
> 
> And it worked for the ten images I tested it with. However, I tested it in a standalone script, so I do not know if this will work well with Fanficfare.

Thanks

**Comment by i-push-code** [comment]
> I'm ill with covid at the moment, can't promise to get to this soon.

Okay, hope you get better soon

**Comment by JimmXinu** [comment]
Fixed in #1004
